compiler/benches/executor_benchmark.rs

```diff
@@ -26,16 +26,11 @@ fn criterion_benchmark(c: &mut Criterion) {
     let contents = compiler::compile(riscv_asm_files, &CoProcessors::base(), false);
     let pil_with_constants = Pipeline::<T>::default()
         .from_asm_string(contents, None)
-        .optimized_pil_with_constants()
+        .pil_with_evaluated_fixed_cols()
```

Comment:
Is this a rebase artifact?

Reply:
No, I renamed the function in this PR to be more consistent with the naming introduced in yours! It returns a `PilWithEvaluatedFixedCols`.